src/commcare_cloud/ansible/roles/pgbouncer/templates/pgbouncer.ini.j2

```diff
@@ -10,7 +10,7 @@
 {% set current_host = inventory_hostname %}
 {% if is_monolith %}{% set current_host = DEFAULT_POSTGRESQL_HOST %}{% endif %}
 {% for db in postgresql_dbs.by_pgbouncer_host.get(current_host, []) %}
-{{ db.name }} = host={{ "localhost" if [db.host] == db.pgbouncer_hosts else db.host }} port={{ postgresql_port }}
+{{ db.name }} = host={{ "localhost" if db.host == inventory_hostname else db.host }} port={{ postgresql_port }}
```

Comment:
@skodde sorry to do this again, but if we accept this change, can you make sure to port this change over to #4454 as well?

Reply:
Just confirming this change was accepted, so we will indeed need to have it ported to your PR @skodde

Reply:
Done, thanks for the heads up!